### PR TITLE
[Fix] Align HSigmoid with pytorch official

### PR DESCRIPTION
**Pull request by teamwong111**
## Motivation

Align `HSigmoid` with pytorch official.
PyTorch `Hardsigmoid` is provided from 1.6.0 and does not support `bias`, `divisor`, `min_value`, `max_value` args.

## Modification

mmcv/cnn/bricks/hsigmoid.py
tests/test_cnn/test_hsigmoid.py

## BC-breaking (Optional)

Yes. Downstream projects could keep compatibility with this PR by modifying the args of `HSigmoid`.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects, like MMDet or MMCls.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by zhouzaida** [comment]
We should note that changing the default value will introduce bc-breaking

File: mmcv/cnn/bricks/hsigmoid.py [hunk 1]
```diff
@@ -8,19 +8,23 @@
...
+        divisor (float): Divisor of the input feature map. Default: 6.0.
         min_value (float): Lower bound value. Default: 0.0.
         max_value (float): Upper bound value. Default: 1.0.
 
     Returns:
         Tensor: The output tensor.
     """
 
-    def __init__(self, bias=1.0, divisor=2.0, min_value=0.0, max_value=1.0):
+    def __init__(self, bias=3.0, divisor=6.0, min_value=0.0, max_value=1.0):
```
Comment (zhouzaida) [inline mmcv/cnn/bricks/hsigmoid.py:27]: We need to print warning information that the default value has been changed.

Comment (teamwong111) [reply mmcv/cnn/bricks/hsigmoid.py:27]: Done.

File: tests/test_cnn/test_hsigmoid.py [hunk 2]
```diff
@@ -9,21 +9,8 @@ def test_hsigmoid():
     with pytest.raises(AssertionError):
         HSigmoid(divisor=0)
 
-    # test with default parameters
-    act = HSigmoid()
```
Comment (zhouzaida) [inline tests/test_cnn/test_hsigmoid.py:13]: We can test `HSigmoid(1, 2, 0, 1)`

**Comment by zhouzaida** [comment]
It's strange that CIs keep getting canceled.

File: mmcv/cnn/bricks/hsigmoid.py [hunk 3]
```diff
@@ -8,20 +10,30 @@
 class HSigmoid(nn.Module):
     """Hard Sigmoid Module. Apply the hard sigmoid function:
     Hsigmoid(x) = min(max((x + bias) / divisor, min_value), max_value)
-    Default: Hsigmoid(x) = min(max((x + 1) / 2, 0), 1)
+    Default: Hsigmoid(x) = min(max((x + 3) / 6, 0), 1)
+
+    Note:
+        In MMCV v1.4.3, we modified the default value of args to align with
```
Comment (zhouzaida) [inline mmcv/cnn/bricks/hsigmoid.py:16]:
```suggestion
        In MMCV v1.4.4, we modified the default value of args to align with
```

File: mmcv/cnn/bricks/hsigmoid.py [hunk 4]
```diff
@@ -8,20 +10,30 @@
...
 
     Returns:
         Tensor: The output tensor.
     """
 
-    def __init__(self, bias=1.0, divisor=2.0, min_value=0.0, max_value=1.0):
+    def __init__(self, bias=3.0, divisor=6.0, min_value=0.0, max_value=1.0):
         super(HSigmoid, self).__init__()
+        warnings.warn(
+            'In MMCV v1.4.3, we modified the default value of args to align '
```
Comment (zhouzaida) [inline mmcv/cnn/bricks/hsigmoid.py:32]:
```suggestion
            'In MMCV v1.4.4, we modified the default value of args to align '
```

File: tests/test_cnn/test_hsigmoid.py (hunk 2 above)
Comment (teamwong111) [reply tests/test_cnn/test_hsigmoid.py]: Done

File: mmcv/cnn/bricks/hsigmoid.py (hunk 3 above)
Comment (teamwong111) [reply mmcv/cnn/bricks/hsigmoid.py]: Done

File: mmcv/cnn/bricks/hsigmoid.py (hunk 4 above)
Comment (teamwong111) [reply mmcv/cnn/bricks/hsigmoid.py]: Done

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**